spinalcordtoolbox/scripts/sct_straighten_spinalcord.py

```diff
@@ -143,11 +143,11 @@ def get_parser():
     optional.add_argument(
         "-param",
         metavar=Metavar.list,
-        help="R|Parameters for spinal cord straightening. Separate arguments with ','."
-             "\nprecision: [1.0,inf[. Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with hanning fitting method. Default=2"
-             "\nthreshold_distance: [0.0,inf[. Threshold at which voxels are not considered into displacement. Increase this threshold if the image is blackout around the spinal cord too much. Default=10"
-             "\naccuracy_results: {0, 1} Disable/Enable computation of accuracy results after straightening. Default=0"
-             "\ntemplate_orientation: {0, 1} Disable/Enable orientation of the straight image to be the same as the template. Default=0",
+        help="R|Parameters for spinal cord straightening. Separate arguments with \",\".\n"
+             "  - precision: [1.0,inf] Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with hanning fitting method. Default=2\n"
```

Comment:
Infinity cannot be included (not a number), so the proper notation is the following:
```suggestion
             "  - precision: [1, inf) Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with Hanning fitting method. Default=2\n"
```

Reply:
Just a question since I don't know. What's the reasoning for changing from a float to an int? Won't there be a loss of precision due to this?

Reply:
the code has not changed from float to int-- only the usage was updated for cleaner display. Many of our users do not understand that displaying the decimal (eg 1.0 instead of 1) suggests that the input will be treated as Float. However, you are right that "float" input should be explicitly mentioned, so I've now added it.